common-custom-user-data-gradle-plugin/src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -301,7 +309,7 @@ private static void addCustomValueAndSearchLink(BuildScanExtension buildScan, St
         }
     }
 
-    private void captureTestParallelization() {
+    private static void captureTestParallelization(Gradle gradle, BuildScanExtension buildScan) {
```

Comment:
is this a separate fix for config cache compatibility?

Reply:
Yes, although it is in a separate commit from the other changes. I might split it out into it's own PR though so that we can get it merged. I was just being lazy.